eox_hooks/actions_handler.py

```diff
@@ -37,17 +41,28 @@ def action_lookup(module_name, action):
         module = import_module(module_name)
         return getattr(module, action)
     except ImportError:
-        message = "The module {} with the action {} does not exist. A default action will be used."\
-                  .format(
-                      module_name,
-                      action,
-                    )
+        message = "The module {} with the action {} does not exist. A default action will be used.".format(module_name,
```

Comment:
ugly
![image](https://user-images.githubusercontent.com/36200299/89210146-2f59c380-d585-11ea-8135-1be55f566f8c.png)

Reply:
I fixed it :100: